wrapper_test.go

```diff
@@ -1,22 +1,110 @@
 package wrapper
 
 import (
+	"fmt"
 	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
 	"testing"
 )
 
+func setup(t *testing.T) *ComposeWrapper {
+	w, err := NewComposeWrapper("")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	return w
+}
+
 func TestCommand(t *testing.T) {
-	w := NewComposeWrapper()
+	w := setup(t)
 
 	file := "docker-compose-test.yml"
-	_, err := w.Up(file)
+	_, err := w.Up(file, "", "", "")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	_, err = w.Down(file, "", "")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+}
+
+const composeFile = `version: "3.9"
+services:
+  busybox:
+    image: "alpine:latest"
+    container_name: "compose_wrapper_test"`
+const composedContainerName = "compose_wrapper_test"
+
+type composeOptions struct {
+	filePath    string
+	url         string
+	envFile     string
+	projectName string
+}
+
+func createFile(dir, fileName, content string) (string, error) {
+	filePath := filepath.Join(dir, fileName)
+	f, err := os.Create(filePath)
 	if err != nil {
-		log.Fatalln(err)
+		return "", err
 	}
 
-	_, err = w.Down(file)
+	f.WriteString(content)
+	f.Close()
+
+	return filePath, nil
+}
+
+func createEnvFile(dir, envFileContent string) (string, error) {
+	return createFile(dir, "stack.env", envFileContent)
+}
+
+func createComposeFile(dir, composeFileContent string) (string, error) {
+	return createFile(dir, "docmer-compose.yml", composeFileContent)
+}
+
+// func Test_UpAndDown(t *testing.T) {
```

Comment:
@chiptus should we try to get those tests active or remove them?

Reply:
actually TestCommand is almost the same, I'll rename it and add checks for container